discovery-provider/src/models/models.py

```diff
@@ -402,14 +402,15 @@ class Playlist(Base):
     txhash = Column(String, default="", nullable=False)
     playlist_id = Column(Integer, nullable=False)
     playlist_owner_id = Column(Integer, nullable=False)
-    is_album = Column(Boolean, nullable=False)
-    is_private = Column(Boolean, nullable=False)
-    playlist_name = Column(String)
-    playlist_contents = Column(JSONB, nullable=False)
-    playlist_image_multihash = Column(String)
-    playlist_image_sizes_multihash = Column(String)
-    description = Column(String)
-    upc = Column(String)
+    metadata_multihash = Column(String, nullable=True)
```

Comment:
making fieldz nullable so everything can go in metadat@

Reply:
are we missing an associated migration for these changes? otherwise models will mismatch actual dB state